### ISSUE TITLE
Ingest ALKIS-Tatsächliche Nutzung

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#156]
## User story
1. As a User
2. I want the map for Tatsächliche Nutzung (actual use)
3. So that I can access the actual use a lot for a specific coordinate/address

## Acceptance criteria
* Ingest and access: https://geodaten.bayern.de/opengeodata/OpenDataDetail.html?pn=tatsaechlichenutzung
* #157 
* #158 
* #159 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
